cd/Jenkinsfile_release_job

```diff
@@ -42,8 +42,8 @@ pipeline {
     // Using string instead of choice parameter to keep the changes to the parameters minimal to avoid
     // any disruption caused by different COMMIT_ID values chaning the job parameter configuration on
     // Jenkins.
-    string(defaultValue: "mxnet_lib", description: "Pipeline to build", name: "RELEASE_JOB_TYPE")
-    string(defaultValue: "cpu,native,cu100,cu101,cu102,cu110", description: "Comma separated list of variants", name: "MXNET_VARIANTS")
+    string(defaultValue: "mxnet_lib/static", description: "Pipeline to build", name: "RELEASE_JOB_TYPE")
```

Comment:
should be mxnet_lib, we removed "/static" in a previous commit https://github.com/apache/incubator-mxnet/commit/dd4661a12230584dc206b4bf233ede261fa64786#diff-dd43bbf192e508d18e340337cf5a6094e137fba710759718cfbde6cf38e27a54R45